less/common/Tooltip.less

```diff
@@ -1,72 +1,209 @@
 // ------------------------------------
 // Tooltips
+// ------------------------------------
 
-// Base class
-.tooltip {
-  position: absolute;
-  z-index: @zindex-tooltip;
-  display: block;
-  font-size: 12px;
-  font-weight: normal;
-  line-height: 1.4;
-  opacity: 0;
-  .transition(0.15s opacity linear);
-
-  &.in     { opacity: 1; }
-  &.top    { margin-top:  -3px; padding: @tooltip-arrow-width 0; }
-  &.right  { margin-left:  3px; padding: 0 @tooltip-arrow-width; }
-  &.bottom { margin-top:   3px; padding: @tooltip-arrow-width 0; }
-  &.left   { margin-left: -3px; padding: 0 @tooltip-arrow-width; }
+// This fixes an issue in iOS where `:hover` is not handled
+[data-device-os="iOS"] {
```

Comment:
We already have a tool for checking mobile safari, I don't see a reason to include the generic getOperatingSystem script beyond that. Plus a more generic tool opens the possibility of special cases, which would be preferable to avoid when possible

Reply:
Except the mobile Safari tool only checks for mobile Safari, not other "browsers" which just embed Safari within their own UI.

If I only used the mobile Safari check, then Chrome for iOS, Firefox for iOS, etc would break.

Reply:
Then we need to modify the mobile safari check, as issues with mobile safari will also occur on those other browsers. That being said, have we confirmed it doesn't hold for chrome ios?

Reply:
When I researched about that specific iOS issue with absolute positioning, I found out it was mobile safari only, but I obviously haven't confirmed that.

That said, seeing how we only apply a `cursor: pointer` here, and it's the only place we use an iOS check, couldn't we just drop the iOS check and apply it for all devices, since I don't think it'll affect other devices.

Reply:
But then we'd be left with weird pointers all over the place, which would look weird, like this: 

![image](https://user-images.githubusercontent.com/7406822/111307734-c4f81e00-8651-11eb-93b0-1886db16e5cb.png)

Reply:
Ah I see, this seems related to https://github.com/flarum/core/pull/2607/files I think we might be able to directly use the same media query ? (can't test..)

Reply:
Hmmm maybe that would work, yeah. I'll give it a go, but I have no iOS devices to test with.

Reply:
I'll try with https://app.lambdatest.com/ when possible and confirm, I've used it before, it's not bad